### ISSUE TITLE
Move Azuracast server (installation)

### ISSUE THREAD

**Issue by JustIrinel** [AzuraCast/AzuraCast#667]
Hi, there is possible to move my server from a provider to another ? How can I do that so all the settings for azuracast remain ? I plan to move on a dedicated server.

**Comment by SlvrEagle23** [comment]
@JustIrinel Are you on a Docker installation or the Traditional VPS installation?

The Docker installation is meant to be easily portable, and you can back up your station files, databases and statistics all into one single file by running `./docker.sh backup`, then moving over to your new server and running `./docker.sh restore [backupfilename.tar.gz]`.

Currently, there is no simple way of accomplishing this with the Traditional installation.

**Comment by JustIrinel** [comment]
Hi @SlvrEagle23  ! Thanks for this great software.

My server is a Traditional Installation. I will need to build from scratch ?

**Comment by SlvrEagle23** [comment]
@JustIrinel You can always just run the traditional installer on the new server, then copy station files over via SFTP and transfer the database contents over with MariaDB's built-in `mysqldump` program. These require somewhat more advanced knowledge of the terminal than is typical with AzuraCast, but it's absolutely possible to do.

**Comment by pwrj** [comment]
> ./docker.sh restore

Do I have install docker or Azuracast in the server first before restoring? I'm getting this: 
root@myradio:~# ./docker.sh restore backup.tar.gz
-bash: ./docker.sh: No such file or directory

**Comment by SlvrEagle23** [comment]
@pwrj You will at least need to install the Docker utility script (which provides you with `docker.sh`); that's the first few lines in the installer:

```bash
mkdir -p /var/azuracast
cd /var/azuracast
curl -fsSL https://raw.githubusercontent.com/AzuraCast/AzuraCast/master/docker.sh > docker.sh
chmod a+x docker.sh
```

**Comment by pwrj** [comment]
@SlvrEagle23 Thank you. It worked like a charm. I restore the back up 100%, now what do I after? Do I stop the old server? What do I need to do to the new server in order to run my radio?

**Comment by pwrj** [comment]
@SlvrEagle23 I got it to work. Stop all instances of docker from old server. Thanks!!

**Comment by radiorueda** [comment]
Hi guys. I am really stucked into this, doing the whole thing for second time. I rented a new server B to move my station, and followed this steps: installation of docker + azuracast. backup in the older server A without media files. restoring backup in B. Copying all the files from A under /var/lib/docker/volumes/azuracast_station_data/_data to the same folder in B. But when I started the services in the new server, no one of the audio files is listed in the playlists, and shoutcast is not starting. Which information should I provide you to solve this? thanks a lot!
